src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritItemListenerTest.java

```diff
@@ -171,18 +172,53 @@ public void testOnJobDeleted() throws Exception {
     }
 
     /**
-     * Subscribes the job to the specified event.
-     * @param job the job.
-     * @param event the event.
+     * Test {@link GerritSaveableListener#onChange(hudson.model.Saveable, hudson.XmlFile)} to handle removed trigger.
+     *
      * @throws Exception if so.
      */
-    public void subscribeJobToEvent(FreeStyleProject job, PatchsetCreated event) throws Exception {
+    @Test
+    public void testOnJobUpdated() throws Exception {
```

Comment:
This is a little naughty, ideally, since I introduce a new listener, I should also introduce a new test class, however, I took this shortcut since there is an significant amount of boilerplate that needs to be copied from this class. If you wan't I can introduce a new class or rename this one.